### ISSUE TITLE
givemoney command

### ISSUE THREAD

**Issue by xenfovn** [zfbx/zdiscord#10]
pls add givemoney command for qb-core

**Comment by zfbx** [comment]
Yeah this is on the roadmap https://github.com/zfbx/zdiscord/projects/1#card-68864856 just been swamped with other things I need to handle. I'll try and add it tonight

**Comment by zfbx** [comment]
I completely forgot about this, I'm so sorry, I've been swamped, I'll try to get it done soon if someone doesn't beat me to it, it's not that difficult of a task really, just copy another similar command that takes an idea then a value and change it to call the addMoney for a player from qb

**Comment by zfbx** [comment]
https://github.com/zfbx/zdiscord/blob/djs/commands/qb-money.js done for the update, can be used now but does require newest fivem artifacts cause it's a rewrite using discord.js
